core-feature-pack/galleon-common/src/main/resources/configs/standalone/model.xml

```diff
@@ -15,6 +15,7 @@
         <prop name="--internal-remove-config" value=""/>
     </props>
     <packages>
+        <package name="banner" optional="true"/>
```

Comment:
My initial concern here was more about how this will affect WildFly full. If this package were enabled in full, we would get the WildFly Core banner in WildFly full. I don't know what are the plans for a banner in WildFly full, if there is one, I guess it will overwrite the banner.txt with its banner and it will use the same package.

Reply:
We can use a WildFly banner instead

Reply:
Yes the idea is to provide the same package with a WildFly Full banner

Reply:
@bstansberry Do yo agree with that ?

Reply:
https://github.com/wildfly/wildfly/pull/17703

Reply:
https://issues.redhat.com/browse/WFLY-19116